examples/wc-dapp-react/src/components/HelloWorld.tsx

```diff
@@ -32,7 +33,7 @@ function HelloWorld() {
 
   const getMyBalance = async (): Promise<void> => {
     try {
-      const resp = await wcSdk.testInvoke({
+      const resp = await wcSdk.withContext(context).testInvoke({
```

Comment:
You've replaced all the calls to use `withContext` but I think we shouldn't do this.
This makes it look like the context message is mandatory, and since it's optional, it will not be used always. Let's simply create a new test method that contains a hardcoded context message to be used as example.